cmd/semaphore/generate/openapi3/openapi.go

```diff
@@ -74,15 +82,20 @@ func run(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	bb, err := yaml.Marshal(object)
-	if err != nil {
-		return err
+	object.Info = openapi3.Info{
+		Title:       "Public API",
+		Version:     cmd.Version,
+		Description: "generated by Semaphore",
+		Contact: &openapi3.Contact{
+			Name:  "Jexia",
+			URL:   "www.jexia.com",
+			Email: "semaphore@jexia.com",
```

Comment:
Let's not include this information inside the open-source repo. It is probably better to point them to the repo instead.